app/src/main/java/com/example/fizzbuzz/fizzbuzz.kt

```diff
@@ -1,5 +1,70 @@
 package com.example.fizzbuzz
 
+fun addWord(words: MutableList<String>, word: String) {
```

Comment:
I like this! Separating your logic out in to nice easy-to-read functions is a really neat approach.